ribs/visualize.py

```diff
@@ -163,12 +165,13 @@ def grid_archive_heatmap(archive,
                       **pcm_kwargs)
 
     # Create the colorbar.
-    if cbar == True:
-        print("Plotting cbar on default figure axis")
-        ax.figure.colorbar(t, ax=ax, pad=0.1)
+    cbar_kwargs = {} if cbar_kwargs is None else cbar_kwargs
+    if cbar == "auto":
+        ax.figure.colorbar(t, ax=ax, **cbar_kwargs)
     elif isinstance(cbar, axes.Axes):
-        print("Plotting cbar on provided figure axis")
-        cbar.figure.colorbar(t, ax=cbar, pad=0.1)
+        cbar.figure.colorbar(t, ax=cbar, **cbar_kwargs)
+    elif cbar is not None:
+        raise ValueError("cbar arg must be 'auto', None, or a matplotlib Axes object")
```

Comment:
Can you move this error message to the beginning of the function? (i.e. create a similar if statement at the beginning) We want to check early because `pcolormesh` may take a while to run, and if this fails, then the user will have waited a long time only to see an error message.

Reply:
When you make this change, also add a small test that checks that an error is thrown e.g. if you pass `cbar='foobarbaz'`